src/main/java/com/ays/assignment/model/dto/request/AssignmentSearchRequest.java

```diff
@@ -13,9 +14,11 @@
  * transfer of the data between different layers of the application.
  */
 @Getter
+@Setter
 @Builder
 public class AssignmentSearchRequest {
 
-    private Double latitude;
     private Double longitude;
+    private Double latitude;
```

Comment:
İki alan için de `@NotNull` anotasyonu kullanmamız daha sağlıklı olabilir

Reply:
Güncellendi! @agitrubard